indy_common/test/auth/test_auth_rev_reg_def_with_new_auth_map.py

```diff
@@ -0,0 +1,20 @@
+from indy_common.authorize.auth_actions import AuthActionAdd, AuthActionEdit
+from indy_common.constants import REVOC_REG_DEF
+
+
+def test_rev_reg_def_adding(write_request_validation, req, is_owner):
+    authorized = req.identifier in ("trustee_identifier", "steward_identifier", "trust_anchor_identifier")
+    assert authorized == write_request_validation(req,
+                                                  [AuthActionAdd(txn_type=REVOC_REG_DEF,
+                                                                 field='some_field',
+                                                                 value='some_value',
+                                                                 is_owner=is_owner)])
+
+
+def test_rev_reg_def_editing(write_request_validation, req, is_owner):
+    assert not write_request_validation(req,
```

Comment:
Please rather check in what cases it's true (the same condition as in previous test, that is either Trustee, Steward or TrustAnchor AND isOwner?)